src/main/resources/db/changelog/changes/04-crate-user-role-table.yaml

```diff
@@ -0,0 +1,39 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-user-roles-table
+      author: coder
+      preConditions:
+        onFail: MARK_RAN
+        conditions:
+          - not:
+              tableExists:
+                tableName: user_roles
```

Comment:
what for?

Reply:
the project was not built because the table already exists